src/app/treasure-hunt/treasure-hunt-report/opportunity-summary/opportunity-summary-copy-table/opportunity-summary-copy-table.component.ts

```diff
@@ -14,15 +16,23 @@ export class OpportunitySummaryCopyTableComponent implements OnInit {
   settings: Settings;
 
   @ViewChild('copyTable', { static: false }) copyTable: ElementRef;
+  @ViewChild('copyTable2', { static: false }) copyTable2: ElementRef;
+  @ViewChild('exportModal', { static: false }) public exportModal: ModalDirective;
   tableString: any;
 
+  fileName: string;
   individualOpportunitySummaries: Array<OpportunitySummary>;
-  constructor() { }
+  constructor(private cd: ChangeDetectorRef) { }
```

Comment:
Try this --> remove ChangeDetectorRef, line 28, and ngAfterViewInit(). Then call this.setInvidiualSummaries() in nGOnInit().